docs/scripts/util/crawlAllLinksFromAllPages.ts

```diff
@@ -0,0 +1,41 @@
+import puppeteer from 'puppeteer';
+import { runArrayPromiseInOrder } from '.';
+
+export async function crawlAllLinksFromAllPages(allPages: string[]) {
+  type AllPagesPaths = Map<
+    string,
+    {
+      pageUrl: string;
+      links: { href: string; tagName: string; tagText: string }[];
+    }
+  >;
+  const allPagesPaths: AllPagesPaths = new Map();
+  await runArrayPromiseInOrder(allPages, checkSitemapPath);
+
+  return allPagesPaths;
+
+  async function checkSitemapPath(pageUrl: string, pageIdx: string) {
```

Comment:
Let's declare this outside of `crawlAllLinksFromAllPages`

Reply:
I did that on purpose to avoid an unnecessary closure